nixos/modules/services/misc/adaptivemm.nix

```
@@ -0,0 +1,86 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.adaptivemm;
+in {
+  options = {
+    services.adaptivemm = {
+      enable = mkEnableOption (lib.mdDoc "Proactively tune kernel free memory configuration");
```

```suggestion
      enable = mkEnableOption "Proactively tune kernel free memory configuration";
```
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557